src/ui.ts

```diff
@@ -1,5 +1,66 @@
 import { IconPicture } from '@codexteam/icons';
 import { make } from './utils/dom';
+import { ImageToolConfig } from './index';
+import type { API } from '@editorjs/editorjs';
+import { ImageToolData } from './index';
+
+enum UiState {
+  EMPTY = "empty",
+  UPLOADING = 'uploading', 
+  FILLED = 'filled'
+};
```

Comment:
The uppercasing is not used in [Enums](https://www.typescriptlang.org/docs/handbook/enums.html).